charts/openebs/latest/templates/daemonset-ndm.yaml

```diff
@@ -8,6 +8,8 @@ metadata:
     release: {{ .Release.Name }}
     heritage: {{ .Release.Service }}
     component: ndm
+    openebs.io/component-name: ndm
```

Comment:
can we use `apps/v1` of the DaemonSet for k8s v1.16 capability support.

Reply:
Made a change.